explainer/async_fn_in_dyn_trait/generalizing_from_box_to_dynx.md

```diff
@@ -176,4 +176,4 @@ impl<O> Drop for dynx Future<Output = O> {
         }
     }
 }
-```
\ No newline at end of file
+```
```

Comment:
🤔 Hmm, I made this PR from the web UI; I don't really know what the change on this line is about 😅